keps/sig-apps/2232-jobs-stopped/README.md

```diff
@@ -222,7 +238,7 @@ the scheduler.
 1. Stopping an active Job will delete *all* active pods.
```

Comment:
what are the mitigations?

I believe we can only offer documentation and best practices for both points.

Reply:
why is this a risk, isn't this something we want by design?

Reply:
It might be a risk depending on how the Pods are stopped and the completion semantics.
It needs to be clarified in docs.

Reply:
sure, it needs to be clarified, but risks are unintended side effects, "delete all active pods" is not an unintended side effect, it is a design choice. So the risk here could be "unknown effects from deleting active pods when stopping a job"

Reply:
unless of course we know the effects or impact on semantics, in which case we should mention them.